docs/source/support/faq.rst

```diff
@@ -2,6 +2,12 @@
 Frequently Asked Questions
 ==========================
 
+Is there a Wiki?
+----------------
+
+Yes - https://github.com/nucypher/wiki/wiki
```

Comment:
This is currently a private repo but I think it needs a small update and to be opened to community contributions. It's better than non-NuCypherinos attempting to update RTD

Reply:
Added as part of #1728 but now deleted (https://github.com/nucypher/nucypher/pull/1728#discussion_r382870665). We can always add it once the relevant updates are made.